src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -673,8 +677,86 @@ def readImage(self):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
-    ## TODO:
-    ## Implementation of uploadImage()
+    def uploadImage(self, imageLocation):
+    """
+    Upload the image of a finger from host computer.
+
+    @param string imageLocation
```

Comment:
Better name the argument `imageFilePath` to make it more clear what has to be passed to the method.